test-files/soa/soa5_gateways_routers/src/org/jboss/soa/esb/samples/quickstart/helloworld/MyJMSListenerAction.java

```diff
@@ -50,7 +50,7 @@ public Message playWithMessage(Message message) throws Exception {
     }
 
     private void logHeader() {
-        System.out.println("\n&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&");
+        System.out.println(System.lineSeparator() &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&");
     }
```

Comment:
Huh?

Reply:
are you scary from test source file? I guess it was taken from some example - not really matter if bad or good ;)